wiki/Gameplay/Input_device/Touch_device/de.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Touchgerät
+
+*Siehe auch: [Spielweise](/wiki/Spielweise)*
+
+Ein **Touchgerät** ist ein Gerät mit Touchscreen, das es dem Benutzer ermöglicht, mit den Fingern auf dem Computer zu arbeiten. osu! unterstützt Touchgeräte von Haus aus. Durch Antippen des Bildschirms kannst du gleichzeitig zielen und klicken. Touchgeräte werden meist in Form von Touchscreen-Laptops, hochwertigeren Grafiktabletts oder Smartphones angeboten.
+
+Wenn ein Touchgerät verwendet wird, um eine osu!-Spielmodus Map zu spielen, wird automatisch ein Mod hinzugefügt, der die Verwendung eines Touchgeräts anzeigt.
```

Comment:
```suggestion
Wenn ein Touchgerät verwendet wird, um eine Map von osu! zu spielen, wird automatisch eine Mod hinzugefügt, der die Verwendung eines Touchgeräts anzeigt.
```

`eine Mod` because of `Modifikation`?

Reply:
The article styling criteria mention that articles shouldn't include terms like `osu!standard` and should use `osu!` instead. That's why I'm suggesting the change above.